### PR TITLE
Close shiny session on window close

### PR DESCRIPTION
**Pull request by jmbuhr**
Adds one line so that shiny doesn't complain when the window is closed.

### CODE REVIEW

File: R/app_server.R [hunk 1]
```diff
@@ -8,6 +8,7 @@
   questions_df <- .get_questions()
 
   output$questions <- .render_questions(questions_df)
+  session$onSessionEnded(stopApp)
```
Comment (r4ds) [inline R/app_server.R]:
```suggestion
  session$onSessionEnded(shiny::stopApp)
```

Comment (r4ds) [reply R/app_server.R]: That needs to be namespaced, right?

**Review by ghost — CHANGES_REQUESTED** [review]

File: R/app_server.R (hunk 1 above)
Comment (jmbuhr) [reply R/app_server.R]: When I execute run_app, I get the message "Loading required package: shiny".
Shiny seems to be loaded automatically with any shiny app.

Comment (r4ds) [reply R/app_server.R]: Technically it's loaded, but it... doesn't feel right for a package function to call an un-namespaced function. For example, as I refine testing, shiny might NOT be loaded in a test context.

Comment (jmbuhr) [reply R/app_server.R]: But I do not quite understand why. There is no `#' @import ` anywhere. It might be a `golem` thing.

**Review by ghost — APPROVED** [review]

**Pull request merged**